types/haversine/index.d.ts

```diff
@@ -30,28 +32,39 @@ declare namespace haversine {
     type Coordinate = (CoordinateLongitudeLatitude | CoordinateLonLat | CoordinateLatLng | LatLonTuple | GeoJSON);
 
     interface Options {
-        /**
-         * Unit of measurement applied to result. Default: "km".
-         */
+        /** Unit of measurement applied to result. Default: "km". */
         unit?: 'km' | 'mile' | 'meter' | 'nmi';
         /**
-         * If passed, will result in library returning boolean value of whether or not the start and end points are within that supplied threshold. Default: null.
+         * If passed, will result in library returning boolean value of whether or not the start and end points are within that supplied threshold.
```

Comment:
I've changed it to `number | null` but it's unclear to me in what sense the default was ever `null`. This looks more like the original author not understanding the distinction between `null` and `undefined`.

Reply:
The original author was me :wink: So I just overlooked the part of `null` because of just copy and pasting the JSDoc strings from the original repository